src/contexts/shimmer/ShimmerContext.tsx

```diff
@@ -55,18 +55,23 @@ const ShimmerProvider = memo(({ children }: Props) => {
 const Container = styled.div`
   position: relative;
   width: 100%;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
 `;
 
 type VisibleProps = { visible: boolean };
 
 const StyledShimmerComponent = styled.div<VisibleProps>`
   position: absolute;
-  opacity: ${({ visible }) => (visible ? 1 : 0)};
+  display: ${({ visible }) => (visible ? 'block' : 'none')};
```

Comment:
the fact that the shimmer still existed on the screen affected children; should now be removed from DOM with display none